types/activation.go

```diff
@@ -135,11 +135,11 @@ func (t *ActivationTx) Id() AtxId {
 	return *t.id
 }
 
-func (t ActivationTx) ShortId() string {
+func (t *ActivationTx) ShortId() string {
```

Comment:
why is it needed? Why is it part of this PR?

Reply:
I spoke to Noam, it was probably made by mistake so I fixed it.
@noamnelke